homeassistant/components/tplink_omada/coordinator.py

```diff
@@ -91,6 +93,24 @@ async def poll_update(self) -> dict[str, OmadaGateway]:
         return {self.mac: gateway}
 
 
+class OmadaDevicesCoordinator(OmadaCoordinator[OmadaListDevice]):
+    """Coordinator for generic device lists from the controller."""
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        config_entry: ConfigEntry,
+        omada_client: OmadaSiteClient,
+    ) -> None:
+        """Initialize my coordinator."""
+        super().__init__(hass, omada_client, "DeviceList", POLL_CLIENTS)
+        self._config_entry = config_entry
```

Comment:
```suggestion
    """Coordinator for generic device lists from the controller."""

    config_entry: ConfigEntry

    def __init__(
        self,
        hass: HomeAssistant,
        config_entry: ConfigEntry,
        omada_client: OmadaSiteClient,
    ) -> None:
        """Initialize my coordinator."""
        super().__init__(hass, omada_client, "DeviceList", POLL_CLIENTS)
```
After `super().__init__()`, `self.config_entry` is automatically set to the config entry

Reply:
not sure where you use it

Reply:
Yeah, sorry, the previous commit had some of the device removal processing inside the coordinator. I removed it later but left this behind. It's removed now.